lib/ansible/plugins/doc_fragments/kubevirt_vm_options.py

```diff
@@ -38,4 +38,66 @@ class ModuleDocFragment(object):
                as a disk to the virtual machine. A proper cloud-init installation is required inside the guest.
                More information U(https://kubevirt.io/api-reference/master/definitions.html#_v1_cloudinitnocloudsource)"
         type: dict
+    affinity:
+        description:
+            - "Describes node affinity scheduling rules for the vm."
+        type: dict
+        version_added: 2.8.2
```

Comment:
Not sure what the rules for this are. 2.9 for now, then change to 2.8.2 if a backport does get merged would be my guess.

Reply:
I would need to then change back the version in devel branch. Let's see what maintainers say.

Reply:
This should be `2.9` as we don't backport features.

Reply:
Thanks. I've seen you've did some exceptions, so I've thought it's now permitted in case there is  changelog. But OK, I will move to 2.9/

Reply:
Exceptions shoud be handled case by case discussed in an IRC meeting. Feel free to put this on the agenda https://github.com/ansible/community/issues/474.

Reply:
Aha, thanks for the info. This PR isn't that urgent.